pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -513,7 +506,7 @@ self: super: {
       };
     in
     super.fruzzy.overrideAttrs (old: {
-      buildInputs = [ nim ];
+      buildInputs = [ nim1 ];
```

Comment:
This is the core changes, all other is just formatting changes via `nixpkgs-fmt`